easybuild/framework/easyconfig/parser.py

```diff
@@ -45,6 +45,59 @@
 # alternate easyconfig parameters, and their non-deprecated equivalents
 ALTERNATE_PARAMETERS = {
     # <new_param>: <equivalent_param>,
+    'build_deps': 'builddependencies',
+    'build_in_install_dir': 'buildininstalldir',
+    'build_opts': 'buildopts',
+    'build_stats': 'buildstats',
+    'clean_up_old_build': 'cleanupoldbuild',
+    'clean_up_old_install': 'cleanupoldinstall',
+    'configure_opts': 'configopts',
+    'deps': 'dependencies',
+    'doc_paths': 'docpaths',
+    'doc_urls': 'docurls',
+    'dont_create_install_dir': 'dontcreateinstalldir',
+    'exts_class_map': 'exts_classmap',
+    'exts_default_class': 'exts_default_class',
+    'exts_default_opts': 'exts_default_options',
+    'hidden_deps': 'hiddendependencies',
+    'include_modulepath_exts': 'include_modpath_extensions',
+    'install_opts': 'installopts',
+    'keep_previous_install': 'keeppreviousinstall',
+    'keep_symlinks': 'keepsymlinks',
+    'max_parallel': 'maxparallel',
+    'env_mod_aliases': 'modaliases',
+    'env_mod_alt_soft_name': 'modaltsoftname',
+    'modulepath_prepend_paths': 'moddependpaths',
+    'env_mod_extra_paths_append': 'modextrapaths_append',
+    'env_mod_extra_paths': 'modextrapaths',
+    'env_mod_extra_vars': 'modextravars',
+    'env_mod_load_msg': 'modloadmsg',
+    'env_mod_lua_footer': 'modluafooter',
+    'env_mod_tcl_footer': 'modtclfooter',
+    'env_mod_class': 'moduleclass',
+    'env_mod_depends_on': 'module_depends_on',
+    'env_mod_force_unload': 'moduleforceunload',
+    'env_mod_load_no_conflict': 'moduleloadnoconflict',
+    'env_mod_unload_msg': 'modunloadmsg',
+    'only_toolchain_mod_env': 'onlytcmod',
+    'os_deps': 'osdependencies',
+    'post_install_cmds': 'postinstallcmds',
+    'post_install_msgs': 'postinstallmsgs',
+    'post_install_patches': 'postinstallpatches',
+    'pre_build_opts': 'prebuildopts',
+    'pre_configure_opts': 'preconfigopts',
```

Comment:
```suggestion
    'pre_config_opts': 'preconfigopts',
```

Reply:
As discussed, let's stick to `pre_configure_opts` (to align with `configure` step)